test.cpp

```diff
@@ -0,0 +1,11 @@
+#define CATCH_CONFIG_MAIN  // This tells Catch to provide a main() - only do this in one cpp file
```

Comment:
Good attempt !
But improvement could still be mage.
For example the requirement is as follows,
in the range 2,3, 3, 5, 4, 10, 11, 12 => we have 2 consecutive ranges, i.e. 2-5 and 10-12
num of elements in 2-5 = 5
num of elements in 10-12 = 3

Reply:
@prithamvshetty : did you mean to increase the test cases ? covering check of all cases ?
I have adapted now , please check & let me know

Reply:
Currently the implementation is such that it only gets the number of values in a given range. where range is a subset of input values.
But if you see the input values, there could be many consecutive ranges. The requirement is to detect all those consecutive ranges and number of values withing each range.
The test cases need to define that. understanding the requirement right is the first step towards quality code 👍